go.mod

```diff
@@ -25,18 +25,18 @@ require (
 	github.com/spf13/viper v1.7.0
 	github.com/stretchr/testify v1.7.0
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5
-	go.opencensus.io v0.22.3
+	go.opencensus.io v0.23.0
```

Comment:
(non-blocking) I wonder why opencensus incremented? Although it doesn't likely matter.

Reply:
🤷 

It looks like the [v0.23.0 was released in Feb 2021](https://github.com/census-instrumentation/opencensus-go/releases/tag/v0.23.0) and has a few bug fixes relative to v0.22.3 (including a memory leak).